ui/src/track-layout/track-layout-slice.ts

```diff
@@ -592,3 +628,102 @@ function getLayoutContext(
         return officialMainLayoutContext();
     }
 }
+
+export const toolPanelAssetExists = (selection: Selection, asset: ToolPanelAsset): boolean => {
```

Comment:
Näistä tuli aivan hirveitä, pääasiassa johtuen meidän `Selection`:in rakenteesta. Tämä funktio siis vaan tutkii löytyykö selectionin sisältä valittua assettia